launcher/launcher/main.go

```diff
@@ -28,6 +30,10 @@ const (
 	// panic() returns 2
 	rebootRC = 3 // reboot
 	holdRC   = 4 // hold
+	// hostTokenPath defined the directory in the host that will store attestation tokens and experiment data.
+	hostTokenPath = "/tmp/container_launcher/"
```

Comment:
This is duplicated with container_runner.go. Any reason we don't pull it out into some constants or common package

Reply:
please rebase off of https://github.com/google/go-tpm-tools/pull/356